packages/firmware-release-config/package.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "@trezor/firmware-release-config",
+    "version": "1.0.0",
+    "private": true,
+    "license": "See LICENSE.md in repo root",
+    "sideEffects": false,
+    "main": "src/index",
+    "scripts": {
+        "build:lib": "yarn validate-release-config && yarn sign-release-config",
+        "depcheck": "yarn g:depcheck",
+        "lint:js": "yarn g:eslint '**/*.{ts,tsx,js}'",
+        "test:unit": "yarn g:jest",
```

Comment:
gives me. how come it doesn't fail in the CI 🤔 
<img width="803" alt="image" src="https://github.com/user-attachments/assets/b0ca49e7-031a-40d8-881e-5bccadcc1ecf" />

Reply:
The CI uses nx that runs `build:lib` for this package before.
It is configure here: https://github.com/trezor/trezor-suite/pull/18435/files#diff-15552e50b1b7c05b05b7ffe08ee47b5ed62b8d2039229c58972a42fd22a7381cR28